package.json

```diff
@@ -27,10 +26,11 @@
   },
   "homepage": "https://github.com/SteamingMutt/DiscordFeedback#readme",
   "devDependencies": {
-    "eslint": "^4.7.0",
+    "eslint": "^4.8.0",
+    "ajv": "^5.2.3",
```

Comment:
?

Reply:
See the comment of the commit that adds it, it’s a new required peer dep for eslint.

Reply:
At least sort it correctly then, it's all in alphabetical order.

Reply:
👍